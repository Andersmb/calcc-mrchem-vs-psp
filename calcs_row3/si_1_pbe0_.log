Starting job 5688422 on c2-[34,51],c5-[29,40-42] at Sat May 21 10:36:51 CEST 2022

Copying si_1_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/triplets
Copying si_1_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/triplets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688422      si_1_pbe0_        180                                             00:43:54      0:0 
5688422.bat+      batch         36        1   00:00:02          0   00:00:02   00:43:54      0:0 
5688422.ext+     extern        180        6   00:00:00          1   00:00:00   00:43:54      0:0 
5688422.0      mrchem.x        180       30   00:43:36         29   02:03:51   00:43:43      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688422                                                                          
5688422.bat+     19332K          0     19332K        0              0          0 
5688422.ext+          0          1          0        0              1          0 
5688422.0     11505408K         29 4664779707        0              2          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688422                                                                                               
5688422.bat+       16.91M               0         16.91M        0.03M                0          0.03M 
5688422.ext+        0.00M               1          0.00M            0                1              0 
5688422.0           6.75M               1          6.73M       70.93M               11         53.81M 

Job 5688422 completed at Sat May 21 11:20:45 CEST 2022
