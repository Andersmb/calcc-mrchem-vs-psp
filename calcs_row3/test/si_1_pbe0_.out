                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               3
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Si            :        0.000000        0.000000       -2.122162
    1      Si            :        0.000000        0.000000        2.122162
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              15
 Beta electrons          :                                              13
 Total electrons         :                                              28
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              1.000000361801e+00
    1    1    a          :                              9.999997986540e-01
    2    1    a          :                              1.000000624890e+00
    3    1    a          :                              9.999997337377e-01
    4    1    a          :                              9.999997561309e-01
    5    1    a          :                              1.000000115747e+00
    6    1    a          :                              1.000004773783e+00
    7    1    a          :                              9.999950412478e-01
    8    1    a          :                              9.999999110936e-01
    9    1    a          :                              1.000000090732e+00
   10    1    a          :                              9.999999611297e-01
   11    1    a          :                              9.999999957839e-01
   12    1    a          :                              9.999999933912e-01
   13    1    a          :                              9.999998784371e-01
   14    1    a          :                              9.999999966254e-01
   15    1    b          :                              1.000000306574e+00
   16    1    b          :                              9.999996562103e-01
   17    1    b          :                              9.999998756708e-01
   18    1    b          :                              1.000000152410e+00
   19    1    b          :                              9.999994801562e-01
   20    1    b          :                              1.000000174774e+00
   21    1    b          :                              1.000004800238e+00
   22    1    b          :                              9.999936424245e-01
   23    1    b          :                              9.999999085846e-01
   24    1    b          :                              1.000000093707e+00
   25    1    b          :                              9.999999601998e-01
   26    1    b          :                              9.999999960817e-01
   27    1    b          :                              9.999999941509e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                572.980979891668
 E-N energy              :            (au)              -1462.671257321017
 Coulomb energy          :            (au)                309.234494409555
 Exchange energy         :            (au)                -10.101400253165
 X-C energy              :            (au)                -31.168370660395
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 46.179314698357
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -621.725553933353
 Nuclear energy          :            (au)                 46.179314698357
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.755462392350e+02
                         :      (kcal/mol)             -3.611607149673e+05
                         :        (kJ/mol)             -1.511096432404e+06
                         :            (eV)             -1.566141032990e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -65.853883234469
    1    1    a          :            (au)                -65.850916142349
    2    1    a          :            (au)                 -5.347041829771
    3    1    a          :            (au)                 -5.345447716306
    4    1    a          :            (au)                 -3.682396282752
    5    1    a          :            (au)                 -3.682203417361
    6    1    a          :            (au)                 -3.681196481915
    7    1    a          :            (au)                 -3.681101781632
    8    1    a          :            (au)                 -3.674733080962
    9    1    a          :            (au)                 -3.673650717535
   10    1    a          :            (au)                 -0.568284229016
   11    1    a          :            (au)                 -0.401902692876
   12    1    a          :            (au)                 -0.260065620568
   13    1    a          :            (au)                 -0.249897136449
   14    1    a          :            (au)                 -0.227844127459
   15    1    b          :            (au)                -65.849675578265
   16    1    b          :            (au)                -65.847844780861
   17    1    b          :            (au)                 -5.343115208673
   18    1    b          :            (au)                 -5.340098704346
   19    1    b          :            (au)                 -3.674454599749
   20    1    b          :            (au)                 -3.673393748466
   21    1    b          :            (au)                 -3.673282378955
   22    1    b          :            (au)                 -3.672482009777
   23    1    b          :            (au)                 -3.672052425300
   24    1    b          :            (au)                 -3.671256845020
   25    1    b          :            (au)                 -0.523269794494
   26    1    b          :            (au)                 -0.348096530842
   27    1    b          :            (au)                 -0.211981544837
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -331.681568641003
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          5.291503e+00        -575.546239234996         -5.755462e+02
    1          1.936014e-01        -578.526300553585         -2.980061e+00
    2          6.060499e-02        -578.615002761290         -8.870221e-02
    3          3.290302e-02        -578.628078939162         -1.307618e-02
    4          4.241615e-02        -578.631929022064         -3.850083e-03
    5          2.288319e-02        -578.632823006796         -8.939847e-04
    6          1.364744e-02        -578.633264609022         -4.416022e-04
    7          6.659673e-03        -578.633349459759         -8.485074e-05
    8          3.853907e-03        -578.633373413336         -2.395358e-05
    9          1.928304e-03        -578.633379076695         -5.663359e-06
   10          1.788482e-03        -578.633383325914         -4.249218e-06
   11          1.016097e-03        -578.633384257621         -9.317068e-07
   12          7.222402e-04        -578.633384663520         -4.058995e-07
   13          8.625461e-04        -578.633384921291         -2.577708e-07
   14          1.165505e-03        -578.633385142872         -2.215810e-07
   15          2.876416e-03        -578.633385011671          1.312006e-07
   16          6.104955e-03        -578.633384690034          3.216370e-07
   17          4.732604e-02        -578.633353294724          3.139531e-05
   18          3.370849e-01        -578.632530516383          8.227783e-04
   19          2.880983e-02        -578.633561554995         -1.031039e-03
   20          2.520986e-01        -578.633382413913          1.791411e-04
   21          1.300526e-02        -578.633381893374          5.205388e-07
   22          1.377625e-02        -578.633377464915          4.428459e-06
   23          4.582100e-03        -578.633383213215         -5.748300e-06
   24          1.115670e-03        -578.633383206199          7.015728e-09
   25          1.037670e-03        -578.633383289641         -8.344148e-08
   26          8.191165e-04        -578.633383334051         -4.441040e-08
   27          7.243956e-04        -578.633383359248         -2.519630e-08
   28          8.461591e-05        -578.633383362160         -2.911861e-09
---------------------------------------------------------------------------
                      SCF converged in 28 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               3
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Si            :        0.000000        0.000000       -2.122162
    1      Si            :        0.000000        0.000000        2.122162
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                577.644020030504
 E-N energy              :            (au)              -1471.407941626282
 Coulomb energy          :            (au)                310.359891945810
 Exchange energy         :            (au)                -10.130415000309
 X-C energy              :            (au)                -31.278253410239
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 46.179314698357
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -624.812698060517
 Nuclear energy          :            (au)                 46.179314698357
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.786333833622e+02
                         :      (kcal/mol)             -3.630979271393e+05
                         :        (kJ/mol)             -1.519201728137e+06
                         :            (eV)             -1.574541579745e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -66.287449675453
    1    1    a          :            (au)                -66.287443861617
    2    1    a          :            (au)                 -5.366293738689
    3    1    a          :            (au)                 -5.366108915008
    4    1    a          :            (au)                 -3.702682035552
    5    1    a          :            (au)                 -3.702587719546
    6    1    a          :            (au)                 -3.701478436545
    7    1    a          :            (au)                 -3.700483449499
    8    1    a          :            (au)                 -3.696379213075
    9    1    a          :            (au)                 -3.696287759230
   10    1    a          :            (au)                 -0.554896594191
   11    1    a          :            (au)                 -0.401219771905
   12    1    a          :            (au)                 -0.259009618320
   13    1    a          :            (au)                 -0.245930848490
   14    1    a          :            (au)                 -0.219633569882
   15    1    b          :            (au)                -66.283857854278
   16    1    b          :            (au)                -66.283843700703
   17    1    b          :            (au)                 -5.362343656782
   18    1    b          :            (au)                 -5.362206743435
   19    1    b          :            (au)                 -3.695400509824
   20    1    b          :            (au)                 -3.695316433248
   21    1    b          :            (au)                 -3.694475302983
   22    1    b          :            (au)                 -3.694387949515
   23    1    b          :            (au)                 -3.692272933332
   24    1    b          :            (au)                 -3.691339290889
   25    1    b          :            (au)                 -0.499710237083
   26    1    b          :            (au)                 -0.349974164928
   27    1    b          :            (au)                 -0.208558720217
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -333.701572704218
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000007        0.000039        0.000065
 Magnitude               :            (au)                        0.000076
                         :         (Debye)                        0.000194
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000       -0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :        0.000007        0.000039        0.000065
 Magnitude               :            (au)                        0.000076
                         :         (Debye)                        0.000194
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 43m 39s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
